studybuddies/src/app/student/create-class-student/create-class-student.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-create-class-student',
+  templateUrl: './create-class-student.component.html',
+  styleUrls: ['./create-class-student.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7155411)
```suggestion
  styleUrls: ["./create-class-student.component.css"]
```